packages/generator/internal.d.ts

```diff
@@ -0,0 +1,3 @@
+// eslint-disable-next-line import/no-internal-modules
```

Comment:
Is this file committed on purpose?

Reply:
yes nicer imports in the api mirror is the reason behind this.